README.rst

```diff
@@ -29,7 +29,12 @@ How can I install this plugin?
 ------------------------------
 
 1. Download this repo.
-2. Terminal move to the repo dir and run ``git submodule update --init --recursive``
+2. Terminal move to the repo dir and run the following commands:
+
+    ``git submodule update --init --recursive``
+
+    ``./prepare.sh``
```

Comment:
you mean the way to run the script or the implementation or both not cross-platform? Any suggestions to make them cross-platform?